services/app/src/static/style/antd-override.css

```diff
@@ -7,6 +7,14 @@
   display: none;
 }
 
+.ant-tooltip-arrow::before {
+  background: none !important;
```

Comment:
Try using ```display: none``` here. The less you use `!important` in your css the better.

Reply:
If I remove `!important` from `.ant-tooltip-arrow::before`, the arrow reverts back to the rectangle. If I remove `!important` from `.ant-tooltip-arrow::after`, the arrow becomes white on hover.